indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -20,36 +21,67 @@ import {
 } from 'indico/react/components';
 import {useNumericParam} from 'indico/react/util/routing';
 import {Translate} from 'indico/react/i18n';
+import {useIndicoAxios} from 'indico/react/hooks';
 import {EditableType} from '../../models';
 import StateIndicator from '../../editing/timeline/StateIndicator';
 
 import './EditableList.module.scss';
 
 export default function EditableList() {
-  const list = []; // Replace it with list from server
   const eventId = useNumericParam('confId');
   const {type} = useParams();
-  const [sortBy, setSortBy] = useState('id');
+  const {data, loading: isLoadingEditableList, lastData} = useIndicoAxios({
+    url: editableListURL({confId: eventId, type}),
+    camelize: true,
+    trigger: eventId,
+  });
+
+  let editableList = data || lastData;
+  if (isLoadingEditableList && !lastData) {
+    return <Loader inline="centered" active />;
+  } else if (!editableList) {
+    return null;
+  }
+
+  editableList = [...data];
```

Comment:
What is the point of this?

Reply:
From the backend, data is coming in multiple json objects and I just destructure them to store in a list.

Reply:
But this line of code just shallow-copies a list...

Reply:
Yeah, you are just copying the array.  It is like a noop in this case

Reply:
oh okay, I got it. Thanks for the catch :+1: